### ISSUE TITLE
How to adjust the weight of loss functions

### ISSUE THREAD

**Issue by dh23333** [wasidennis/AdaptSegNet#86]
![image](https://user-images.githubusercontent.com/35715068/87241287-2ed17f80-c454-11ea-98e0-bb481967c649.png)

You said that modifying the model requires fine-tuning the parameters. Is it convenient for you to remind about the tricks about adjusting the weight of loss functions?

**Comment by wasidennis** [comment]
We just try different weights in a reasonable rage, e.g., [5e-3, 2e-3, 1e-3, 5e-4, 2e-4, 1e-4] and so on.
